pwndbg/gdblib/config.py

```diff
@@ -53,7 +55,7 @@ def __init_super_gdb_gte_9(self, param: pwndbg.lib.config.Parameter) -> None:
                 param.name,
                 gdb.COMMAND_SUPPORT,
                 param.param_class,
-                param.enum_sequence,
+                param.enum_sequence if param.enum_sequence else [],
```

Comment:
why?

Reply:
Because `enum_sequence: Sequence[str] | None = None,`
`enum_sequence` can be `None` that's why we need to check there

Reply:
But can it really be None? Is there a case when this actually occurs? That's the problem with some of the  changes here; we don't need to make redundant checks/logic :(

Reply:
And those things are not exactly free, since we are in CPython land which is slow and we also want to give the best possible UX to the user. This may also matter a lot if Pwndbg is used in gdbscripts

Reply:
fixed!